src/shared/components/Footer/Footer.tsx

```diff
@@ -1,34 +1,84 @@
-import React, { FunctionComponent } from 'react';
-import { Link } from 'react-router-dom';
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { RouteComponentProps } from 'react-router-dom';
 
 import { Container } from '@viaa/avo2-components';
+import { Avo } from '@viaa/avo2-types';
+
+import { connect } from 'react-redux';
+import { selectLoginMessage, selectUser } from '../../../authentication/store/selectors';
+import { AppState } from '../../../store';
+import { mapNavElementsToNavigationItems, renderNavLinkItem } from '../../helpers/navigation';
+import {
+	AppContentNavElement,
+	getNavigationItems,
+	NavItemMap,
+} from '../../services/navigation-items-service';
+import { NavigationItem } from '../../types';
+
+import './Footer.scss';
+
+export interface FooterProps extends RouteComponentProps {
+	user: Avo.User.User | undefined;
+}
+
+const Footer: FunctionComponent<FooterProps> = ({ history, location, match, user }) => {
+	const [t] = useTranslation();
+
+	const [areDropdownsOpen, setDropdownsOpen] = useState<{ [key: string]: boolean }>({});
```

Comment:
I recall dat dit op andere plaatsen op dezelfde manier gebruikt wordt, deze interface. Kunnen we dit anders een shared interface maken?